examples/sampling/graphbolt/node_classification.py

```diff
@@ -70,7 +66,7 @@ def create_dataloader(
         The node features.
     itemset : Union[ItemSet, ItemSetDict]
         Data to be sampled.
-    job : Literal["train", "evaluate", "infer"]
+    job : one of ["train", "evaluate", "infer"]
```

Comment:
Is it more felicitous to use `Union` ?

Reply:
Union is more for data type. Literal is the right option, but let's avoid it until more and more users switch to 3.8.